src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -749,7 +749,8 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
 
     rekordbox_anlz_t anlz = rekordbox_anlz_t(&ks);
 
-    double sampleRateFrames = sampleRate * 2.0;
+    double sampleRateScaled = sampleRate / 1000.0;
```

Comment:
Instead of `sampleRateScaled` we could name this variable more explicitly `sampleRateKhz`?

Reply:
My mistake, this actually wasn't a problem. Have reverted.